packages/web-components/fast-components-msft/src/card/card.styles.ts

```diff
@@ -1,7 +1,8 @@
 import { css } from "@microsoft/fast-element";
-import { display } from "@microsoft/fast-foundation";
+import { display, forcedColorsStylesheetBehavior } from "@microsoft/fast-foundation";
 import { elevation } from "../styles";
 import { neutralLayerCardBehavior } from "../styles/index";
+import { SystemColors } from "@microsoft/fast-web-utilities";
```

Comment:
@khamudom I know these are coming in with the other PR, but they should be removed from here as they are currently referenced.

Reply:
shoot. thank you!